### ISSUE TITLE
scp Using "user@host" Fails, but "-l user host" Succeeds

### ISSUE THREAD

**Issue by thetuber** [gravitational/teleport#6688]
[teleport.log](https://github.com/gravitational/teleport/files/6408646/teleport.log)
[tsh.log](https://github.com/gravitational/teleport/files/6408647/tsh.log)

### Description

**What happened**:

Tried to copy a remote file using:

user@machine$ tsh --proxy teleport.example.com:443 --user tuser scp ruser@tptestnode:/etc/issue /tmp
Enter password for Teleport user tuser:
ERROR: node "Node:\"tptestnode\" Login:\"user\" " not found

Using the -l syntax works:

user@machine$ tsh --proxy teleport.example.com:443 --user tuser scp -l ruser tptestnode:/etc/issue /tmp
Enter password for Teleport user tuser:
<- /tmp/issue (26)

### Reproduction Steps
As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
1. Copy a remote file:
tsh --proxy teleport.example.com:443 --user tuser scp ruser@tptestnode:/etc/issue /tmp
2.
3.

### Server Details
- Teleport version (run `teleport version`): Teleport v6.1.2 git:v6.1.2-0-g23ab88c00 go1.15.5
- Server OS (e.g. from `/etc/os-release`):
NAME="Ubuntu"
VERSION="18.04.5 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.5 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

- Where are you running Teleport? (e.g. AWS, GCP, Dedicated Hardware): Azure
- Additional details:

### Client Details
- Tsh version (`tsh version`): Teleport v6.1.2 git:v6.1.2-0-g23ab88c00 go1.15.5
- Computer OS (e.g. Linux, macOS, Windows): GNU Debian 10
- Browser version (for UI-related issues): N/A
- Installed via (e.g. apt, yum, brew, website download): dpkg
- Additional details:

### Debug Logs
Please include or attach debug logs, when appropriate. Obfuscate sensitive information!
- Start Teleport with --debug flag (`teleport --debug`)
- Run tsh with --debug flag (`tsh --debug`)

**Comment by thetuber** [comment]
Suggested PR https://github.com/gravitational/teleport/pull/6690

**Comment by programmerq** [comment]
This has been fixed in #6927

**Comment by butsjoh** [comment]
Sorry to say but in Teleport v6.2.8 git:v6.2.8-0-g0bbeffb63 go1.16.2 we still have to use the -l flag for this to work.

Example that does not work: (details removed of course)

```
 tsh scp /path/to_file_1  user@node:/destination

ERROR: node "Node:\"xxx\" Login:\"xxx\" " not found
```

Example that works: (details removed of course)

```
 tsh scp -l user /path/to_file_1  user@node:/destination
```
